exec/BuildDocs.hs

```diff
@@ -157,15 +157,17 @@ instance ToSample [ThentosSessionToken] [ThentosSessionToken] where
     toSample _ = Just ["abde1234llkjh", "47202sdfsg"]
 
 instance ToSample UserFormData UserFormData where
-    toSample _ = Just $ UserFormData (UserName "Kurt Cobain")
-                                   (UserPass "Hunter2")
-                                   (UserEmail "cobain@nirvana.com")
+    toSample _ = Just $ UserFormData (UserName "Kurt Cobain") (UserPass "Hunter2") sampleEmail
```

Comment:
I would prefer:

```haskell
  toSample _ = do
    sampleEmail <- toSample
    Just $ UserFormData (UserName "Kurt Cobain") (UserPass "Hunter2") sampleEmail
```

...  and then make your `sampleEmail` below a `ToSample` instance.